Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilder.cpp

```diff
@@ -293,6 +293,8 @@ namespace ScriptCanvasBuilder
                 overrideValue.SetAllowSignalOnChange(false);
             }
         }
+
+        m_isComponentScript = abstractCodeModel->GetInterface().RefersToSelfEntityId();
```

Comment:
RefersToSelfEntityId, isComponentScript, and scriptcanvas_component_extension all refer to the new distinction made between graphs that will only function properly when used with a ScriptCanvasComponent on an Entity, as apposed to those that will function properly in other contexts.